src/Control/Middleware/TrustedProxyMiddleware.php

```diff
@@ -1,31 +1,55 @@
 <?php
 
-namespace SilverStripe\Control;
+namespace SilverStripe\Control\Middleware;
+
+use SilverStripe\Control\HTTPRequest;
+use SilverStripe\Control\Util\IPUtils;
 
 /**
  * This middleware will rewrite headers that provide IP and host details from an upstream proxy.
  */
 class TrustedProxyMiddleware implements HTTPMiddleware
 {
-
+    /**
```

Comment:
Case in point, this helped me discover the mismatch between the string to array assignment I noticed earlier. Even though it's not public, it helps with bug-detection.